src/components/Button.js

```diff
@@ -1,11 +1,43 @@
 import React from 'react'
 import styles from './Button.module.css'
 
-export const Button = ({ onClick, children }) => {
+export const FormButton = ({ onClick, children }) => {
   console.log(onClick)
   return (
   <button onClick={onClick} className={`${styles.button} ${styles.primary}`}>
     {children}
   </button>
   )
 }
+
+export const InfoButton = ({ page, children }) => {
```

Comment:
I would change this to href as it seems more obvious to me:

```suggestion
export const InfoButton = ({ href, children }) => {
```